ethsigner/signer/multikey/src/test/java/tech/pegasys/ethsigner/signer/multikey/HashicorpSigningMetadataTomlConfigLoaderTest.java

```diff
@@ -235,10 +62,18 @@ void relativeHashicorpAuthFileIsRelativeToLibraryRoot() throws IOException {
     final HashicorpSigningMetadataFile metadataFile =
         (HashicorpSigningMetadataFile) metadataFiles.toArray()[0];
 
-    assertThat(metadataFile.getConfig().getAuthFilePath())
-        .isEqualTo(configsDirectory.resolve("./path/to/auth-file"));
-    assertThat(metadataFile.getConfig().getTlsKnownServerFile().get())
-        .isEqualTo(configsDirectory.resolve("./path/to/known-server-file"));
+    final HashicorpKeyConfig hashicorpConfig = metadataFile.getConfig();
+
+    assertThat(hashicorpConfig.getConnectionParams().getServerHost()).isEqualTo("Host");
```

Comment:
nit: extract hashicorpConfig.getConnectionParams() to a local variable to shorten the assertations

Reply:
done